noty-android/app/build.gradle

```diff
@@ -77,6 +77,10 @@ dependencies {
     // JetPack Security
     implementation "androidx.security:security-crypto:$securityCryptoVersion"
 
+    // Navigation
+    implementation "androidx.navigation:navigation-fragment-ktx:$navigationVersion"
+    implementation "androidx.navigation:navigation-ui-ktx:$navigationVersion"
+
```

Comment:
What's the need of including this in `app` module? It's already included in `simpleapp` module I guess.

Reply:
See this: https://github.com/PatilShreyas/NotyKT/blob/master/noty-android/app/simpleapp/build.gradle#L125-L126

Reply:
oops will remove this from app module and migrate the extensions function to simpleapp module!